### PR TITLE
docs: add workshop banner

### PR DESCRIPTION
**Pull request by kay-kim**
Will revert on Oct 3 (well, the changes for the height adjustments will be reverted.  will keep the banner.html and related css).

### CODE REVIEW

File: doc/user/config.toml [hunk 1]
```diff
@@ -3,8 +3,11 @@ title = "Materialize Documentation"
 pygmentsCodeFences = true
 pygmentsUseClasses = true
 
+
 [params]
 repo = "//github.com/MaterializeInc/materialize"
+bannerMessage = "Register for the upcoming (2024-Oct-03) workshop: Operational Data Store vs. Read Replicas."
```
Comment (morsapaes) [inline doc/user/config.toml:9]:
```suggestion
bannerMessage = "Register for the upcoming live workshop: Materialize vs. PostgreSQL read replicas. Oct. 3rd, 2PM ET
```

**Review by morsapaes — APPROVED** [review]
What do you think about placing the banner on top of the header, rather than below?

**Comment by kay-kim** [comment]
As for the location of the banner, I actually started with the banner at top 😄 Then, after looking at it for a while, it somehow seemed easy to miss, so moved it to the bottom.  But, I'll admit ... when placed at the top, it looks great at first but after a while, it looks like it should be at the bottom.  Then, when placed at the bottom,  it looks great at first  but after a while seems like it should be at the top.  But I figured we can try different locations as time goes on -- either with this one or with next banner.

**Comment by morsapaes** [comment]
Screenshot 2024-09-06 at 14 57 26

It interferes too much with documentation reading for me.

**Comment by kay-kim** [comment]
> It interferes too much with documentation reading for me.
Oh ... you actually wanted to read the docs 😜 Heh heh ... I'll switch it.

**Comment by kay-kim** [comment]
Added https://github.com/MaterializeInc/materialize/pull/29419 to move banner to top

**Pull request merged**